yt_dlp/extractor/prankcast.py

```diff
@@ -62,5 +62,5 @@ def _real_extract(self, url):
             'cast': list(filter(None, [uploader] + traverse_obj(guests_json, (..., 'name')))),
             'description': json_info.get('broadcast_description'),
             'categories': [json_info.get('broadcast_category')],
-            'tags': self._parse_json(json_info.get('broadcast_tags') or '{}', video_id)
+            'tags': json_info.get('broadcast_tags')
```

Comment:
```suggestion
            'tags': traverse_obj(json_info, ('broadcast_tags', {lambda x: x.split(',') if x else None})),
```
would safely return a list of tags if they exist or else `None` (and tests would need to be updated again)

Reply:
Why not:

```suggestion
            'tags': traverse_obj(json_info, ('broadcast_tags', {lambda x: x.split(',')})
```
or even better:
```suggestion
            'tags': try_call(lambda: json_info['broadcast_tags'].split(','))
```

Reply:
Those work and are simpler, mb. I thought `if x else None` may have been useful if the value of `broadcast_tags` was an empty string, but I guess ignoring that isn't necessary. Edited my suggestion above to avoid confusion